Readme.md

```diff
@@ -0,0 +1,26 @@
+Description
+===========
+
+This obs_lsstSim package provides an interface to the phosim output for the LSST Data Management software.
+
+Updating camera description
+---------------------------
+
+The camera description FITS files are built from the phosim lsst data text files. To update the camera camera description to match a new version of phosim, you will need two files from `phosim_release/data/lsst`, and the gain and saturation data from the header of a phosim generated file.
```

Comment:
camera camera -> camera
header of a phosim generated file -> header of the phosim generated files for the entire focalplane